linuxdeploy-plugin-gtk.sh

```diff
@@ -124,12 +129,7 @@ echo "Copying more libraries"
 gobject_libdir="$("$PKG_CONFIG" --variable=libdir gobject-2.0)"
 gio_libdir="$("$PKG_CONFIG" --variable=libdir gio-2.0)"
 librsvg_libdir="$("$PKG_CONFIG" --variable=libdir librsvg-2.0)"
-cp $verbose \
-    "$gdk_libdir/"libgdk_pixbuf*.so* \
-    "$gobject_libdir/"libgobject*.so* \
-    "$gio_libdir/"libgio*.so* \
-    "$librsvg_libdir/"librsvg*.so* \
-    "$APPDIR/usr/lib/"
-cat >> "$HOOKFILE" <<EOF
-export LD_LIBRARY_PATH="\$APPDIR/usr/lib:\$LD_LIBRARY_PATH"
-EOF
+"$LINUXDEPLOY" --appdir="$APPDIR" --library="$(find "$gdk_libdir"     -type f -name libgdk_pixbuf-*.so*)"
```

Comment:
You should do these deployments in one call to speed up the process.

Reply:
Do you mean I can pass `--library` multiple times?

Reply:
Yep, that's possible. You can also just use `-l mylib`.